tox.ini

```diff
@@ -55,10 +56,20 @@ commands =
     clean: python -c "import shutil, sys; shutil.rmtree(sys.argv[1], ignore_errors=True)" "{toxinidir}/{env:BUILD_DIR}"
     serve: stb serve "{toxinidir}/{env:SOURCE_DIR}/"
 
+[testenv:tests]
+description = Runs the test suite
+dependency_groups =
+    tests
+depends =
+    doc-html
+commands =
+    python -c "print('Running tests...')"
```

Comment:
Let's remove this for the moment until we figure out how to span two processes in Tox at the same time.